VocaDbModel/Domain/Songs/PVForSong.cs

```diff
@@ -119,6 +119,7 @@ public virtual async Task RefreshMetadata(IPVParser pvParser, IUserPermissionCon
 			Author = result.Author;
 			ExtendedMetadata = result.ExtendedMetadata;
 			ThumbUrl = result.ThumbUrl;
+			Length = result.LengthSeconds ?? 0;

```

Comment:
I think it should default to current length (this.Length), if for some reason we could not get length.